todo_app/data/session_items.py

```diff
@@ -54,6 +54,25 @@ def add_item(title):
     return item
 
 
+def remove_item(item):
+    """
+    Removes a single item from the session.
+
+    Args:
+        item: The item to remove.
+    """
+    existing_items = get_items()
+
+    # Remove the item from the list
+    existing_items.remove(item)
+
+    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
```

Comment:
Minor - These two lines seem to be doing very similar things! Either way should be correct but no need to have both 
so:
```
existing_items.remove(item)
session['items'] = existing_items
```

or
```
updated_items = [item for item in existing_items if item["id"] != int(id)]
session['items'] = updated_items
```